lib/puppet.rb

```diff
@@ -33,6 +33,10 @@
   def _(msg)
     msg
   end
+
+  def n_(msg_singular, msg_plural, count)
+    return count == 1 ? msg_singular : msg_plural
```

Comment:
Yep, makes sense. When redefining a method I just try to lean as close to honoring the semantics of the original method as possible. Seems like it ends up being less surprising down the road sometimes. but you're right - what I meant by count was nonsensical because it's not like we're going to be needing to use or supplying additional pluralization rules to support multiple forms. So never mind.